src/main/java/com/thealgorithms/datastructures/trees/nearestRightKey.java

```diff
@@ -3,7 +3,9 @@
 import java.util.Scanner;
 import java.util.concurrent.ThreadLocalRandom;
 
-class Main {
+final class Main {
+    private Main() {
```

Comment:
Could you rename this class?
```suggestion
    private NearestRightKey() {
```